migrate/mysql/000001_initial_schema.up.sql

```diff
@@ -2,3 +2,23 @@ CREATE TABLE service (
   id BINARY(16) PRIMARY KEY,
   name VARCHAR(255)
 );
+
+CREATE TABLE message (
+  id BINARY(16) PRIMARY KEY,
+  service_id BINARY(16) NOT NULL,
+  language VARCHAR(20) NOT NULL,
+
+  FOREIGN KEY (service_id) REFERENCES service (id)
+);
+
+CREATE TABLE message_message (
+  message_id BINARY(16) NOT NULL,
+  message_service_id BINARY(16) NOT NULL,
+  id TEXT NOT NULL,
```

Comment:
Did you miss to remove it?

```suggestion
  message_id BINARY(16) NOT NULL,
  id TEXT NOT NULL,
```

Reply:
Yup it seems so. This field is not used in anywhere except save